src/recordlinker/linkage/matchers.py

```diff
@@ -18,9 +18,9 @@
 SIMILARITY_MEASURES = typing.Literal["JaroWinkler", "Levenshtein", "DamerauLevenshtein"]
 
 # A Callable type for comparing a feature on two records
-FEATURE_COMPARE_FUNC = typing.Callable[[models.PIIRecord, models.Patient, models.FEATURE], bool]
+FEATURE_COMPARE_FUNC = typing.Callable[[models.PIIRecord, models.Patient, models.Feature], float]
```

Comment:
The `models.Patient` class does directly correspond to the `mpi_patient` table in the database, so I would very much like to leave those names matching, just so it's easy for people to make the connection between the model and the table.  I'm open to new names for `models.PIIRecord` though, as that's really just a serializer meant to store and capture PII data to compare.  However, the model is used by BOTH incoming and existing payloads, so naming it `models.IncomingPatient` might be confusing (like when we do something like `patient.incomingpatient.data`).  Maybe there is a better term for labeling the serialized data we want to compare?

Alternatively, we can also just rename the variables in the matcher functions themselves to be more descriptive.

Reply:
Can you ELI5 the purposes of the `models.Patient` and `models.PIIRecord` classes, respectively? How `models.PIIRecord` is used for both incoming and existing payloads? I think this will help inform naming.

Reply:
I'll do my best....

`models.Patient` is intended to store all the Patient information we received in a previous health care document, as well as any calculations we made on the data.  It's a historical record of the document, at least the Patient information within.  A models.Patient, inherently **has** a reference to a models.PIIRecord.

`models.PIIRecord` is a lightweight structure around a collection of patient identifiers.  Having a defined structure around the information makes it easier to compare data elements between different payloads (aka PIRecords).  A models.PIIRecord object **does not know of** a models.Patient.